zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/publishing/PublishNotificationTest.java

```diff
@@ -0,0 +1,268 @@
+package com.github.onsdigital.zebedee.model.publishing;
+
+import com.github.davidcarboni.httpino.*;
+import com.github.onsdigital.zebedee.configuration.Configuration;
+import com.github.onsdigital.zebedee.json.CollectionDescription;
+import com.github.onsdigital.zebedee.json.ContentDetail;
+import com.github.onsdigital.zebedee.json.EventType;
+import com.github.onsdigital.zebedee.model.Collection;
+import org.apache.http.ProtocolVersion;
+import org.apache.http.message.BasicStatusLine;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.mockito.*;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@FixMethodOrder(MethodSorters.JVM) // force name ordering
+public class PublishNotificationTest {
+    @Mock
+    Collection collection;
+    ArrayList<String> urisToUpdate;
+    ArrayList<ContentDetail> urisToDelete;
+    @Captor
+    ArgumentCaptor<Endpoint> endpointCapture;
+    @Captor
+    ArgumentCaptor<PublishNotification.NotificationPayloadCacheApi> payloadCapture;
+    @Captor
+    ArgumentCaptor<Class> classCapture;
+
+    Date publishDate = new Date(1609866000000L);
+    Date clearCacheDateTime;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
+
+        urisToUpdate = new ArrayList<>();
+        urisToUpdate.add("/economy/inflationandprices/bulletins/latest");
+        urisToUpdate.add("/economy/inflationandprices/bulletins/october2022");
+        urisToUpdate.add("/economy/inflationandprices/bulletins/october2023");
+
+        ContentDetail cd1 = new ContentDetail();
+        ContentDetail cd2 = new ContentDetail();
+
+        urisToDelete = new ArrayList<>();
+        urisToDelete.add(cd1);
+        urisToDelete.add(cd2);
+
+        clearCacheDateTime = new DateTime(publishDate).plusSeconds(Configuration.getSecondsToCacheAfterScheduledPublish()).toDate();
+        CollectionDescription cdmock = mock(CollectionDescription.class);
+        when(collection.getDescription()).thenReturn(cdmock);
+        when(collection.getDescription().getId()).thenReturn("cake-1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef");
+        when(collection.getDescription().getPublishDate()).thenReturn(clearCacheDateTime);
+    }
+
+    @Test
+    public void payloadCacheAPIIsCorrect() {
+        try (MockedStatic<Configuration> mocked = mockStatic(Configuration.class)) {
```

Comment:
There is no need to surround this in a try-with-resources. Instead, you can just instantiate it in the `@Before` and then in `@After` you call `close()`. See how we've done it in https://github.com/ONSdigital/babbage/pull/734